src/repr/src/row.rs

```diff
@@ -584,10 +598,10 @@ pub fn datum_size(datum: &Datum) -> usize {
         Datum::Int64(_) => 1 + size_of::<i64>(),
         Datum::Float32(_) => 1 + size_of::<u32>(),
         Datum::Float64(_) => 1 + size_of::<u64>(),
-        Datum::Date(_) => 1 + size_of::<NaiveDate>(),
-        Datum::Time(_) => 1 + size_of::<NaiveTime>(),
-        Datum::Timestamp(_) => 1 + size_of::<NaiveDateTime>(),
-        Datum::TimestampTz(_) => 1 + size_of::<DateTime<Utc>>(),
+        Datum::Date(_) => 1 + 8,
```

Comment:
Does anything stop this from breaking if some future version of chrono changes how many bytes the internal types are?